gengokit/httptransport/templates/server.go

```diff
@@ -188,7 +197,7 @@ func errorEncoder(_ context.Context, err error, w http.ResponseWriter) {
 }
 
 type errorWrapper struct {
-	Error string ` + "`json:\"error\"`\n" + `
+	Error string ` + "`" + `json:"error"` + "`" + `
```

Comment:
I'm familiar with raw strings but have never seen this. The context missing was the open backtick at the very top of the template and the one at the end of the template.